bin/pygrb/pygrb_make_offline_workflow

```diff
@@ -0,0 +1,370 @@
+    return cp
+
+
+def get_coh_PTF_files(cp, ifos, run_dir, bank_veto=False, summary_files=False):
+    """
+    Retrieve the bank_veto_bank.xml file needed to run coh_PTF_inspiral jobs.
+    """
+    if os.getenv("LAL_SRC") is None:
+        raise ValueError("The environment variable LAL_SRC must be set to a "
+                         "location containing the file lalsuite.git")
+    else:
+        lalDir = os.getenv("LAL_SRC")
+        sci_seg = segment(int(cp.get("workflow", "start-time")),
+                          int(cp.get("workflow", "end-time")))
+        file_list = _workflow.FileList([])
+
+        # Bank veto
+        if bank_veto:
+            shutil.copy("%s/lalapps/src/ring/coh_PTF_config_files/" \
+                        "bank_veto_bank.xml" % lalDir, "%s" % run_dir)
+            bank_veto_url = "file://localhost%s/bank_veto_bank.xml" % run_dir
+            bank_veto = _workflow.File(ifos, "bank_veto_bank",
+                                       sci_seg, file_url=bank_veto_url)
+            bank_veto.PFN(bank_veto.cache_entry.path, site="local")
+            file_list.extend(_workflow.FileList([bank_veto]))
+
+        if summary_files:
+            # summary.js file
+            shutil.copy("%s/lalapps/src/ring/coh_PTF_config_files/" \
+                        "coh_PTF_html_summary.js" % lalDir, "%s" % run_dir)
+            summary_js_url = "file://localhost%s/coh_PTF_html_summary.js" \
+                             % run_dir
+            summary_js = _workflow.File(ifos, "coh_PTF_html_summary_js",
+                                       sci_seg, file_url=summary_js_url)
+            summary_js.PFN(summary_js.cache_entry.path, site="local")
+            file_list.extend(_workflow.FileList([summary_js]))
+
+            # summary.css file
+            shutil.copy("%s/lalapps/src/ring/coh_PTF_config_files/" \
+                        "coh_PTF_html_summary.css" % lalDir, "%s" % run_dir)
+            summary_css_url = "file://localhost%s/coh_PTF_html_summary.css" \
+                              % run_dir
+            summary_css = _workflow.File(ifos, "coh_PTF_html_summary_css",
+                                         sci_seg, file_url=summary_css_url)
+            summary_css.PFN(summary_css.cache_entry.path, site="local")
+            file_list.extend(_workflow.FileList([summary_css]))
+
+        return file_list
+
+
+def make_exttrig_file(cp, ifos, sci_seg, out_dir):
+    '''
+    Make ExtTrig xml file
+    '''
+    xmldoc = ligolw.Document()
+    xmldoc.appendChild(ligolw.LIGO_LW())
+    tbl = lsctables.New(lsctables.ExtTriggersTable)
+    xmldoc.childNodes[-1].appendChild(tbl)
+    row = lsctables.ExtTriggersTable()
+    row.process_id = None 
+    row.det_alts = None 
+    row.det_band = None 
+    row.det_fluence = None 
+    row.det_fluence_int = None 
+    row.det_name = None 
+    row.det_peak = None 
+    row.det_peak_int = None 
+    row.det_snr = '' 
+    row.email_time = 0 
```

Comment:
Add the functionality into grb_utils.py as a make_exttrig_file function for now. This might get moved later to a more general make_empty_xml object, but that can wait.